packages/client/src/comp/StateContainer.tsx

```diff
@@ -1,17 +1,39 @@
 import React from 'react'
 import { HashRouter } from 'react-router-dom'
-import { QueryContainer, AddressContainer, ContactContainer, AnalyticsContainer, VoterContainer } from '../lib/unstated'
+import { Slide, ToastContainer } from 'react-toastify'
+import { QueryContainer, AddressContainer, ContactContainer, AnalyticsContainer, VoterContainer, ZipFinderContainer } from '../lib/unstated'
+import 'react-toastify/dist/ReactToastify.css'
+
+const toastPosition = () => {
+  return window.matchMedia('(max-width: 991px)').matches
+    ? 'top-center'
+    : 'bottom-right'
+}
+
 // export for testing purposes
 export const UnstatedContainer: React.FC<{}> = ({ children }) => (<HashRouter>
-  <QueryContainer.Provider>
-    <AddressContainer.Provider>
-      <ContactContainer.Provider>
-        <AnalyticsContainer.Provider>
-          <VoterContainer.Provider>
-            {children}
-          </VoterContainer.Provider>
-        </AnalyticsContainer.Provider>
-      </ContactContainer.Provider>
-    </AddressContainer.Provider>
-  </QueryContainer.Provider>
+  <ZipFinderContainer.Provider>
+    <QueryContainer.Provider>
+      <AddressContainer.Provider>
+        <ContactContainer.Provider>
+          <AnalyticsContainer.Provider>
+            <VoterContainer.Provider>
+              {children}
+              <ToastContainer
```

Comment:
Does `ToastContainer` have to wrap `{children}`?

Reply:
Actually it is supposed to be in this way (when I used this library for the first time that bit surprised me as well).

ToastContainer doesn't render its children, thus we'll always end up with a blank page when passing children inside this Container.